src/dmd/dsymbol.d

```diff
@@ -991,6 +991,19 @@ extern (C++) class Dsymbol : RootObject
         return false;
     }
 
+    /****************************************
+    * extern(C++) needs to know this to mangle properly
+    */
+    final bool isPrimaryDtor()
```

Comment:
@ibuclaw does this need to be added to C++ header?

Reply:
I don't think so...?

Reply:
I can remove this function completely and roll the logic into a simpler one-lineer at each of the call-sites.

Reply:
What about adding a function in `cppmangle`? That can be imported to `cppmanglewin`. Or we wait for Iain.